docs/design/mono/components.md

```diff
@@ -11,6 +11,33 @@ linking).
 To that end this document describes a methodology for defining and implementing components and for calling component
 functionality from the runtime.
 
+## Possible scenarios
+
+The goal is to provide workloads with the ability to ship a runtime that only
+the required capabilities and excludes native code for unsupported operations.
+For example:
+
+- Developer inner loop in on a mobile or WebAssemblyi workload: The runtime
```

Comment:
~~in on a mobile or WebAssemblyi workload~~ -> on a mobile or WebAssembly workload